tools/scripts/install-sdk-preview.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs')
```

Comment:
nit: The script doesn't install the sdk preview. Install in yarn/npm has a meaning of downloading the dependencies. All this script is doing is `setup-npmrc-preview-repository.js` or `setup-gh-repository-to-npmrc.js` or `setup-npmrc-preview.js`